src/Screens/MainPage/components/SearchMainPage.tsx

```diff
@@ -27,38 +36,54 @@ export const SearchMainPage = () => {
   };
   const handleDropDown = (value: string) => {
     console.log(value);
-
+  };
+  const handleInputBlur = () => {
+    setFocus(false);
+  };
+  const handleInputFocus = () => {
+    setFocus(true);
   };
 
   useEffect(() => {
+    addItem();
```

Comment:
it always better that "add" functions will get the parameter they should add and not rely on the state.

addIItem(item)

Also add item to where?
Be more specific with the naming